### PR TITLE
Ff series relay 6

### PR DESCRIPTION
**Pull request by Mrzwanzig**

### CODE REVIEW

File: ffrelaytoolv1/TeamControl.cs [hunk 1]
```diff
@@ -104,30 +105,15 @@ private TabPage createSplitsPage(MetaContext context, TeamInfo info, int tabCoun
...
             for (int i = 0; i < context.splitsToShow; i++)
             {
-                teamSplitNames[i] = Util.createBaseLabel(haMargin, splitLabelHeight * i + context.layout.boxMargin, context.layout.boxWidth - timerWidth - context.layout.boxMargin, splitLabelHeight, "test+" + i);
+                teamSplitNames[i] = Util.createBaseLabel(splitLabelWidth * i + context.layout.boxMargin, context.layout.boxHeight / 2, splitLabelWidth, context.layout.boxHeight / 2 - context.layout.boxMargin, "test+" + i, ContentAlignment.MiddleCenter);
                 tabPageSplits.Controls.Add(teamSplitNames[i]);
-                teamSplitTimes[i] = Util.createBaseLabel(context.layout.boxWidth - timerWidth - haMargin, splitLabelHeight * i + context.layout.boxMargin, timerWidth, splitLabelHeight, "00:00:00");
+                teamSplitTimes[i] = Util.createBaseLabel(splitLabelWidth * i + context.layout.boxMargin, haMargin, splitLabelWidth, context.layout.boxHeight / 2 - context.layout.boxMargin, "00:00:00", ContentAlignment.MiddleCenter);
                 tabPageSplits.Controls.Add(teamSplitTimes[i]);
             }
-            vsLabelNames = new Label[context.numberOfTeams - 1];
```
Comment (MLSTRM) [inline ffrelaytoolv1/TeamControl.cs:117]: This should probably have another option in the file as to what page we want this on

File: ffrelaytoolv1/TeamControl.cs [hunk 2]
```diff
@@ -354,12 +369,22 @@ private void updateVsSplits(VersusWrapper[] otherTeams)
...
 
         private void updateSplits(VersusWrapper[] otherTeams)
         {
-
             int i = Util.clamp(teamInfo.teamSplitNum, context.splits.Length - (context.splitsToShow - context.splitFocusOffset), context.splitFocusOffset);
+            if (context.features.syncSplits)
+            {
+                for (int team = 0; team < otherTeams.Length; team++)
+                {
+                    int j = Util.clamp(otherTeams[team].splitNum, context.splits.Length - (context.splitsToShow - context.splitFocusOffset), context.splitFocusOffset);
```
Comment (MLSTRM) [inline ffrelaytoolv1/TeamControl.cs]: int i = Math.Max(i, Util.clamp(blah blah blah))
Either that or have more descriptive variable names :)

File: ffrelaytoolv1/Util.cs [hunk 3]
```diff
@@ -44,6 +44,14 @@ public static void updateDifferenceDisplay(Label label, TimeSpan seg)
             label.Text = current;
         }
 
+        public static void updateSingleDifferenceDisplay(Label label, TimeSpan seg)
+        {
+            if (seg.TotalHours == 0)
```
Comment (MLSTRM) [inline ffrelaytoolv1/Util.cs]: Should probably be <= 0

File: ffrelaytoolv1/Util.cs [hunk 4]
```diff
@@ -44,6 +44,14 @@ public static void updateDifferenceDisplay(Label label, TimeSpan seg)
             label.Text = current;
         }
 
+        public static void updateSingleDifferenceDisplay(Label label, TimeSpan seg)
```
Comment (MLSTRM) [inline ffrelaytoolv1/Util.cs]: Potentially should be renamed to `UpdatePositiveDifferenceDisplay`

**Review by MLSTRM — CHANGES_REQUESTED** [review]

**Comment by Mrzwanzig** [comment]
Requested changes have been implemented.

**Review by MLSTRM — APPROVED** [review]

**Pull request merged**